cqrs-codegen/impl/src/event/event.rs

```diff
@@ -39,8 +39,6 @@ fn derive_struct(input: syn::DeriveInput) -> Result<TokenStream> {
 /// Implements [`crate::derive_event`] macro expansion for enums
 /// via [`synstructure`].
 fn derive_enum(input: syn::DeriveInput) -> Result<TokenStream> {
-    util::assert_attr_does_not_exist(&input.attrs, super::ATTR_NAME)?;
```

Comment:
Just removing this is not enough. We're just losing some guarantees that way by introducing source of confusion about non-working attributes.

I suggest instead of removing this check make it more deep and verify that concrete sub-attributes are absent.

Reply:
Done.